host/src/default_outgoing_http.rs

```diff
@@ -0,0 +1,42 @@
+use crate::{
+    wasi,
+    wasi::http_types::{
+        Error as HttpError, FutureIncomingResponse as Response, Method, OutgoingRequest as Request,
+        RequestOptions,
+    },
+    WasiCtx,
+};
+
+#[async_trait::async_trait]
+impl wasi::default_outgoing_http::Host for WasiCtx {
+    async fn handle<'a>(
+        &'a mut self,
+        _req: Request,
+        _options: Option<RequestOptions>,
+    ) -> wasmtime::Result<Response> {
+        todo!()
+    }
+}
```

Comment:
rather than panic, can we `anyhow::bail!("not implemented")`? That way it only traps execution